### ISSUE TITLE
 list_attached_tags_on_objects got error with vCenter 6.0 but succeeded with 6.5 an 6.7

### ISSUE THREAD

**Issue by neilcx** [vmware/vsphere-automation-sdk-ruby#48]
## Describe the bug
Try to use list_attached_tags_on_objects `CisTaggingTagAssociationListAttachedTagsOnObjects` to list attached tags on a specified virtual machine. With same code base, it succeeded on vCenter 6.5 and 6.7. But on 6.0, I got 

`VSphereAutomation::ApiError:
       Not Found`

## How to reproduce

prepare a vCenter 6.0 and 6.5 environment, create a vm , category/tag, and attach tag to the vm.
  
```ruby
#!/usr/bin/env ruby

require 'vsphere-automation-cis'
require 'vsphere-automation-vcenter'

configuration = VSphereAutomation::Configuration.new.tap do |config|
  config.host='192.168.111.106'
  config.username='XXX'
  config.password='XXX'
  config.scheme='https'
  config.verify_ssl=false
  config.verify_ssl_host=false
end

api_client = VSphereAutomation::ApiClient.new(configuration)
api_client.default_headers['Authorization'] = configuration.basic_auth_token
session_api = VSphereAutomation::CIS::SessionApi.new(api_client)
session_id = session_api.create('').value
api_client.default_headers['vmware-api-session-id'] = session_id
api_client.default_headers['Cookie'] = "vmware-api-session-id='#{session_id}'"

# try to list attached tags on 'vm-139' 
tag_association_api = VSphereAutomation::CIS::TaggingTagAssociationApi.new(api_client)
object_id_hash = { "type" => "VirtualMachine",  "id" => 'vm-139' }
list_attached_tags_on_object = VSphereAutomation::CIS::CisTaggingTagAssociationListAttachedTagsOnObjects.new(object_ids: [VSphereAutomation::CIS::VapiStdDynamicID.new(object_id_hash)])
list_attached_tags_on_objects_result  = tag_association_api.list_attached_tags_on_objects(list_attached_tags_on_object)

tags_on_vm_info = list_attached_tags_on_objects_result.value[0]
puts tags_on_vm_info
```

## Expected behavior
Expect to list tag info. 

## Versions

**Dev Environment:**

* Ruby: [ruby 2.4.2p198 (2017-09-14 revision 59899) [x86_64-darwin17]]
* vSphere Automation SDK: [0.1.0]
* OS: [macOS 10.13.6 (17G7024)]
* vCenter: 
Name:         VMware vCenter Server
Vendor:       VMware, Inc.
Version:      6.0.0
Build:        7037393
OS type:      linux-x64
API type:     VirtualCenter
API version:  6.0
Product ID:   vpx
UUID:         b78b3a27-92b6-4b54-9418-5184b6f6bebc

## Additional notes
1. production code can be found here, around line 290 in :
https://github.com/cloudfoundry/bosh-vsphere-cpi-release/blob/fd7bdebf12eb9f509d95df3636400055cd77f2e6/src/vsphere_cpi/spec/integration/attach_tag_to_vm_spec.rb

2. CisTaggingTagAssociationListAttachedObjectsOnTags got similar problem.

**Comment by jrgarcia** [comment]
@neilcx It appears that this functionality is not available on vCenter 6.0 from vAPI. That's why it is returning a 404.

**Comment by neilcx** [comment]
@EleanorRigby @yeshwantbabar FYI.
